src/components/codeBlock.tsx

```diff
@@ -9,50 +9,80 @@ import memoize from 'lodash/memoize';
 
 import {useOnClickOutside} from 'sentry-docs/utils';
 
-import {CodeContext} from './codeContext';
+import {CodeContext, createOrgAuthToken} from './codeContext';
 
 const KEYWORDS_REGEX = /\b___(?:([A-Z_][A-Z0-9_]*)\.)?([A-Z_][A-Z0-9_]*)___\b/g;
 
+const ORG_AUTH_TOKEN_REGEX = /__ORG_AUTH_TOKEN__/g;
```

Comment:
This is inconsistently two underscores instead of three like the rest of these keywords

Reply:
I actually did this on purpose to avoid this being captured by the _other_ regex. If we use triple underscore here, we need to make sure checking order is right (IMHO making this a bit more brittle for future refactorings), and it may also not be too clear that this does not go through the same code 🤔 maybe a _completely_ different identifier would be even better, to make it more clear this is something else? For example `+++ORG_AUTH_TOKEN+++`?

Reply:
In that case we should write tests.

If we're going to start having custom behavior for different tokens, we should implement it in a way that it's clear that this component supports handling that.

Right now using a different number of underscores is just going to be **confusing**, since it's really hard to notice (I already fixed a number of places where we were using 2 instead of 3).

I think the way to go here is to implement something a little bit more robust